### PR TITLE
Add information about telemetry in Studio

### PR DESCRIPTION
**Pull request by sdnts**

### CODE REVIEW

**Comment by sdnts** [comment]
Should I also make it clear that their schema / data is never actually sent?

File: content/03-reference/03-more/02-telemetry.mdx [hunk 1]
```diff
@@ -26,7 +26,7 @@ Data is collected in two scenarios that are described below.
 
 ### Usage data
 
-Invocations of the `prisma` CLI sends information to the telemetry server at https://checkpoint.prisma.io. Note that this is only happening at most every 48 hours (i.e., the sending the data to the telemetry server gets paused for 48 hours after any invocation).
+Invocations of the `prisma` CLI & general usage of Studio send information to the telemetry server at https://checkpoint.prisma.io. Note that this is only happening at most every 48 hours (i.e., the sending the data to the telemetry server gets paused for 48 hours after any invocation).
```
Comment (mhwelander) [inline content/03-reference/03-more/02-telemetry.mdx]:
```suggestion
Invocations of the `prisma` CLI and general usage of Studio results in data being sent to the telemetry server at https://checkpoint.prisma.io. Note that:

* Information is only sent every 48 hours (sending is paused for 48 hours after any invocation)
* The data does **not** include your schema or the data in your database
```

**Comment by mhwelander** [comment]
> Should I also make it clear that their schema / data is never actually sent?

@madebysid I added a note about that - will squash-merge if you're happy

**Comment by sdnts** [comment]
Thanks! LGTM!

File: content/03-reference/03-more/02-telemetry.mdx [hunk 2]
```diff
@@ -26,7 +26,10 @@ Data is collected in two scenarios that are described below.
 
 ### Usage data
 
-Invocations of the `prisma` CLI sends information to the telemetry server at https://checkpoint.prisma.io. Note that this is only happening at most every 48 hours (i.e., the sending the data to the telemetry server gets paused for 48 hours after any invocation).
+Invocations of the `prisma` CLI and general usage of Studio results in data being sent to the telemetry server at https://checkpoint.prisma.io. Note that:
+
+* Information is only sent every 48 hours (sending is paused for 48 hours after any invocation)
```
Comment (janpio) [inline content/03-reference/03-more/02-telemetry.mdx]: Don't think this is correct: The duration is now 12 hours @2color @matthewmueller?

Comment (2color) [reply content/03-reference/03-more/02-telemetry.mdx]: Yes. It's 12 hours.

Comment (mhwelander) [inline content/03-reference/03-more/02-telemetry.mdx]:
```suggestion
* Information is only sent every 12 hours (sending is paused for 12 hours after any invocation)
```

Comment (mhwelander) [reply content/03-reference/03-more/02-telemetry.mdx]: Fixed - thanks!

**Pull request merged**